Documenting tool integration (#104)
jsh9/pydoclint: docs/how_to_ignore.md

# How to ignore certain violations

> ## With native tool

Reviewer: ```suggestion
## As a native tool
```

> Currently, this not implemented. Please use _flake8_ mode, or feel free to

Reviewer: ```suggestion
Currently, this is not implemented. Please use _flake8_ mode, or feel free to
```

> Currently, this is not implemented. Please use _flake8_ mode, or feel free to

Reviewer: ```suggestion
Currently, pydoclint does not support ignoring certain violations as a native tool. Please use it as a _flake8_ plugin to achieve that, or feel free to
```

Author: Nice suggestion, implemented

contribute this feature.

> ## With _flake8_ mode

Reviewer: ```suggestion
## In _flake8_ mode
```

> ## In _flake8_ mode

Reviewer: ```suggestion
## As a _flake8_ plugin
```

> # How to ignore certain violations

Author: @jsh9 I am removing this `in _flake8_ mode` because the `index.md` doesn't have it, and this page is now talking about both possible routes

Reviewer: Got it.  Yeah this sounds good

All the usage is consistent with how you would use _flake8_. Please read the
official _flake8_ documentation for full details:
https://flake8.pycqa.org/en/latest/user/violations.html.

> ### Usage with `ruff`

Reviewer: ```suggestion
### Usage with [`ruff`](https://github.com/astral-sh/ruff)
```

> ### Usage with [`ruff`](https://github.com/astral-sh/ruff)

Reviewer: ```suggestion
### Usage with [`Ruff`](https://github.com/astral-sh/ruff)
```

Author: I would rather have one of these two:

- Ruff (no code highlighting)
- `ruff` (matching command line)

I will move to "Ruff" not "`Ruff`"

With `ruff>=0.1.3`, allowlist `DOC` codes using the
[`external` setting](https://docs.astral.sh/ruff/settings/#external):

> 

Reviewer: ```suggestion

Put the following in your `pyproject.toml` file:

```

```toml

> # pyproject.toml

Reviewer: ```suggestion
```
